raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -130,7 +130,8 @@ contract TokenNetwork is Utils {
 
     event NonClosingBalanceProofUpdated(
         uint256 indexed channel_identifier,
-        address indexed closing_participant
+        address indexed closing_participant,
```

Comment:
Why is this attribute named `closing_participant`? The closer is exactly the one node that cannot update the balance proof. Can't we name it just `participant`, `counterparty` or something else?

Reply:
The event is
```
event NonClosingBalanceProofUpdated(
        uint256 indexed channel_identifier,
        address indexed closing_participant,
        uint256 nonce
    );
```
My thinking was: we are updating the balance data for the `closing_participant`. The `nonce` is also for the `closing_participant`'s balance proof.

About naming, please comment on https://github.com/raiden-network/raiden-contracts/issues/201 (`Additional Ideas` contains some naming suggestions; regardless of choosing simple names rather than more suggestive ones, we should have consistency -> we need to agree on something). 

So, your suggestion here is to have:
```
event NonClosingBalanceProofUpdated(
        uint256 indexed channel_identifier,
        address indexed participant,
        uint256 nonce
    );
```
As, it should be clear what role `participant` has. Correct?

Reply:
> As, it should be clear what role participant has. Correct?

I don't think it's necessarily clear who's address it is, but my first thought was "closing_participant is wrong, participant is less wrong".

But now I understand your rationale, this is a hard one to name, if I get a good idea I write in the linked issue.